README.md

```diff
@@ -87,6 +87,15 @@ url = Aws::CF::Signer.sign_url 'http://d604721fxaaqy9.cloudfront.net/training/or
                                ip_range: '145.168.143.0/24'
 ```
 
+With multiple resources
+
+```ruby
+url = Aws::CF::Signer.sign_url 'http://d604721fxaaqy9.cloudfront.net/training/orientation.avi',
+                               expires: 'Sat, 14 Nov 2009 22:20:00 GMT',
+                               resource: [ 'http://d604721fxaaqy9.cloudfront.net/training/*', 'http://d604721fxaaqy9.cloudfront.net/lessons/*' ],
+                               ip_range: '145.168.143.0/24'
+```
+
```

Comment:
What's the use case here?

Why not _http://d604721fxaaqy9.cloudfront.net/*_?

In ["Creating a Policy Statement for a Signed URL That Uses a Custom Policy"](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-creating-signed-url-custom-policy.html#private-content-custom-policy-statement), it says:

> Note the following:
> * You can include only one statement.